tests/storage/test_transfer.py

```diff
@@ -100,8 +100,17 @@ def test_transfer_from_remote_to_local():
     assert artifact2.organisms.get(name="mouse") == bt.settings.organism
     assert artifact.features["obs"].get(name="organism").uid == "existing"
 
+    # test transfer form a remote instance with less schema modules to a local instance (laminlabs/lamin-site-assets)
```

Comment:
It should be called "source instance" and "fewer schema modules than target instance".

"local" and "remote" aren't relevant concepts here.